runtime/src/main/java/io/quarkiverse/cxf/transport/VertxServletOutputStream.java

```diff
@@ -20,7 +20,7 @@
 
 public class VertxServletOutputStream extends ServletOutputStream {
 
-    private HttpServerRequest request;
+    private final HttpServerRequest request;
     protected HttpServerResponse response;
     private ByteBuf pooledBuffer;
     private long written;
```

Comment:
Fields `written` and `finished` are written to but never read which means they can be removed. Let me know if you want me to do this.